src/assets/translations/en/main.json

```
@@ -278,13 +278,15 @@
       },
       "foxFarmingOverview": {
         "header": "Deposit ETH-FOX LP tokens to %{opportunity} to earn",
-        "body": "Approve and stake your liquidity tokens to earn bonus FOX rewards.",
+        "body": "Approve and stake your FOX-ETH UniswapV2 liquidity pool tokens tokens to earn bonus FOX rewards.",
```

We may also want with "LP tokens" for conciseness 
```suggestion
        "body": "Approve and stake your ETH/FOX UniswapV2 liquidity pool tokens tokens to earn bonus FOX rewards.",
```